samples/XCT.Sample/Pages/TestCases/MediaElementSourcePage.xaml.cs

```diff
@@ -1,27 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Xamarin.CommunityToolkit.Core;
+﻿using Xamarin.CommunityToolkit.Core;
 using Xamarin.Forms;
-using Xamarin.Forms.Xaml;
 
 namespace Xamarin.CommunityToolkit.Sample.Pages.TestCases
 {
 	public partial class MediaElementSourcePage
 	{
-		public MediaElementSourcePage()
-		{
-			InitializeComponent();
-		}
+		public MediaElementSourcePage() => InitializeComponent();
 	}
 
-
 	class MediaElementViewModel : BindableObject
 	{
 		public string VideoAsString { get; set; } = "https://tipcalculator.appwithkiss.com/video/Hint_1_2_EN_12.mov";
 
-		public MediaSource VideoAsMediaSource => MediaSource.FromUri(VideoAsString);
+		public MediaSource? VideoAsMediaSource => MediaSource.FromUri(VideoAsString);
```

Comment:
`FromUri` returns `MediaSource?`

Reply:
I mean, that the value that we pass to FromUri method isn't null, we fill it with a value in the class initialization.  Even with that, the return can be null? (sincerely asking because I didn't know how the method is implemented)

Reply:
I chose to keep `FromUri(Uri? uri)`, because there are Unit Tests written around handing cases when `Uri` is null

Reply:
Here's the Unit Test in `MediaSource_Tests` that was testing its nullability.

Because this unit test already existed, I assumed that the author of 1MediaSource` wanted to support a null `Uri`.

```csharp
[Fact]
public void TestImplicitUriConversionWhenNull()
{
	Uri? u = null;
	var sut = (Core.MediaSource?)u;
	Assert.Null(sut);
}
```

Reply:
I think the best way to move forward it to keep this method:
 ```
public MediaSource? VideoAsMediaSource => MediaSource.FromUri(VideoAsString);
```

Then after we merge the PR, the author of `MediaSource` can the chose if they want to support `Uri?` or only support `Uri`.

Reply:
@brminnick I believe that we can let this be an `Uri?` thanks for the explanation.